util/os.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	iscsiutil "github.com/longhorn/go-iscsi-helper/util"
+)
+
+const (
+	OsReleasePath = "/etc/os-release"
+)
+
+func GetHostKernelRelease() (string, error) {
+	initiatorNSPath := iscsiutil.GetHostNamespacePath(HostProcPath)
+	mountPath := fmt.Sprintf("--mount=%s/mnt", initiatorNSPath)
+	output, err := Execute([]string{}, "nsenter", mountPath, "uname", "-r")
+	if err != nil {
+		return "", err
+	}
+	return RemoveNewlines(output), nil
+}
+
+func GetHostOSDistro() (string, error) {
+	initiatorNSPath := iscsiutil.GetHostNamespacePath(HostProcPath)
+	mountPath := fmt.Sprintf("--mount=%s/mnt", initiatorNSPath)
+	output, err := Execute([]string{}, "nsenter", mountPath, "cat", OsReleasePath)
```

Comment:
Can make this reusable, since it is repeated to use in different host info collection.